com.unity.mobile.notifications/Runtime/iOS/Plugins/UnityNotificationData.h

```diff
@@ -0,0 +1,87 @@
+//
+//  UnityNotificationData.h
+//  iOS.notifications
+//
+
+#if TARGET_OS_IOS
+
+#ifndef UnityNotificationData_h
```

Comment:
why not #pragma once?

Reply:
Generated by xcode automatically, since others are using `Include guard`, I followed the convention...